work/Maya/exercise4_1.py

```diff
@@ -0,0 +1,16 @@
+import os
+import sys
+
+def display_permission(file_path):
+    if not os.path.exists(file_path):
+        print(f"{file_path} does not exist")
+        return
+    permission = os.stat(file_path).st_mode
+    print(f"permission of {file_path} is {permission}")
+    print(f"Write permission: {'Yes' if permission & 0o200 else 'No'}")
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("Need the file path as an argument")
```

Comment:
**Note:** Consider allowing the user to input the file path interactively if no argument is provided.